sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/properties/AADAuthenticationProperties.java

```diff
@@ -120,20 +116,30 @@
     private boolean allowTelemetry = true;
 
     /**
-     * If <code>true</code> activates the stateless auth filter {@link AADAppRoleStatelessAuthenticationFilter}. The
-     * default is <code>false</code> which activates {@link AADAuthenticationFilter}.
+     * If true activates the stateless auth filter {@link AADAppRoleStatelessAuthenticationFilter}. The
+     * default is false which activates {@link AADAuthenticationFilter}.
```

Comment:
should we also remove the {@link } here?

Reply:
@Why?

Reply:
Updated.